spec/models/wizards/signup/sektion_operation_spec.rb

```diff
@@ -77,6 +77,47 @@
     expect(max.phone_numbers.first.number).to eq "+41 79 123 45 67"
   end
 
+  it "#save! doesnt create invoice when approval is needed" do
+    expect { operation.save! }
+      .to not_change { ExternalInvoice::SacMembership.count }
+      .and not_change { Delayed::Job.where('handler like "%CreateInvoiceJob%"').count }
+  end
+
+  context "no approval" do
+    subject(:operation) { described_class.new(person_attrs: person_attrs, group: groups(:bluemlisalp_neuanmeldungen_nv), register_on:, newsletter:) }
+
+    # destroy sektion neuanmeldung group to trigger no approval needed case
+    before { group.really_destroy! }
+
+    it "#save! creates invoice and starts job" do
```

Comment:
Kannst du den Test vielleicht umschreiben sodass `register_on` nicht `today` ist? 
Weil eigentlich wäre ja die assumption dass issued_at & sent_at `= Date.current` und nicht `today` sind.
Zudem ist (zumindest was ich dem Ticket entnehme) `register_on` immer am ersten vom nächsten Monat (oder?) Daher etwas verwirrend dass das hier im Test grundsätzlich `today` sein kann
Am besten auch noch in einen context block damits gleich strukturiert ist wie auf Zeile 106